src/index.html

```diff
@@ -20,8 +21,476 @@
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav header__nav">
+          <li>
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <div class="card__content">
+            <div class="card__image">
+              <img
+                src="/src/images/imac.jpeg"
+                width="160px"
+                height="134px"
+                alt="Imac"
+              />
+            </div>
+
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+
+            <p class="card__code">Product code: 195434</p>
+
+            <div class="card__rating">
+              <div
+                class="stars"
+                data-stars="4"
+              >
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <p class="card__label">Price:</p>
+              <p class="card__cost">$2,199</p>
+            </div>
+
+            <a
+              href="#"
+              data-qa="card-hover"
+            >
+              <button class="card__button card__button--hover">Buy</button>
+            </a>
+          </div>
+        </div>
+        <div
```

Comment:
Add the blank line between the cards
```suggestion
        </div>
        
        <div
```